### ISSUE TITLE
Should be able to delete my Surge account

### ISSUE THREAD

**Issue by monfresh** [sintaxi/surge#90]
It would be great to add a command similar to `teardown` that will remove my email address from Surge.

**Comment by o1dnik** [comment]
@monfresh agree. Voting +1 for this. I shouldn't have tried this service before realizing I can't delete an account O_o ...

**Comment by matthieubellon** [comment]
I shouldn't have either, so use to have that kind of "feature"

**Comment by sintaxi** [comment]
I like the idea of having this built into the CLI. Obviously nuking accounts is a delicate thing so we prefer to be cautious with such a potentially destructive command.

The fastest way to have your account removed is to run `surge teardown <domain>` on all your projects then email `support@surge.sh` to request account removal.

FWIW - surge.sh has never sent out newsletters, promotional emails, or shared email addresses with a 3rd party.

**Comment by matthieubellon** [comment]
`surge list` does not display my domain names but `[object Object]` so I can't have their names and teardown them. But I sure know this is not important and also know that I want my account removed definitively.

Not insinuating you are doing bad stuff *today* with our emails, but nobody know what's the future is made off (somebody buy your company, get access to all databases etc ..)

I just deleted my account in seconds from Zeit/Now, they warned me, I confirmed, done. 

Users sure make mistakes, but if they are warned properly, it is on them and no fingers should be pointed to you.

**Comment by sintaxi** [comment]
To clarify, if you would like your account removed email `support@surge.sh` from your surge email address. Commenting on this issue will not be interpreted as a request to have your account removed. 

Tearing down your domains ahead of time will make the process faster but is not necessary.

**Comment by matthieubellon** [comment]
Yes sure, I was not clear, should have wrote = "In my usecase, with the bug I am facing, having access to a delete my account would be great"

I'll send an email, thanks

**Comment by y0y0z** [comment]
@sintaxi: truly love surge.sh service, but the only thing that still bugs me is this issue. could you please share when do you see the issue being addressed?

**Comment by bonki** [comment]
This means that test accounts created using throwaway email addresses cannot currently be deleted and will remain stale as there's no way to actually *send* mail from those. There are also other, more legitimate use cases where it's impossible or not feasible to send mail from the same address which was used for sign-up (e.g. catch-all addresses, ad hoc aliases (i.e. via plus addressing), ....). I consider services which simply assume that I can send mail - especially when not telling me in advance and/or providing the possibility to change my email address - "broken". This also means I cannot easily exercise my right to be forgotten. Now I'm sure you would be willing to help nonetheless, but it's an unnecessary hoop one should not have to jump through.

TL;DR: Please build this into the client!

PS: Sorry if this sounds ranty. I absolutely loved the first impression of surge.sh and this unnecessarily leaves a bitter aftertaste.

**Comment by sintaxi** [comment]
@bonki You have convinced me. I'll add `surge nuke` to the CLI as soon as possible. :)
